# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoadRequest.cpp
# Commit: f0d873f64979
# Full Hash: f0d873f64979928a059b519ea8409dd3c5286d39
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-10-03 20:53 +0000
# Regressor Bug: 1968890
# File Overlap Count: 5
# Overlapping Files: js/loader/ModuleLoadRequest.cpp, js/loader/ModuleLoaderBase.cpp, js/loader/ModuleLoadRequest.h, js/loader/ModuleLoaderBase.h, dom/script/ScriptLoader.cpp
# Description:
#   Bug 1968890 - Part 7: Add Cancel method. r=jonco
# ==============================================================================

diff -r f06186536887 -r f0d873f64979 js/loader/ModuleLoadRequest.cpp
--- a/js/loader/ModuleLoadRequest.cpp	Fri Oct 03 20:53:29 2025 +0000
+++ b/js/loader/ModuleLoadRequest.cpp	Fri Oct 03 20:53:29 2025 +0000
@@ -82,22 +82,6 @@
   return mModuleScript->HasParseError() || mModuleScript->HasErrorToRethrow();
 }
 
-void ModuleLoadRequest::Cancel() {
-  if (IsCanceled()) {
-    return;
-  }
-
-  if (IsFinished()) {
-    return;
-  }
-
-  ScriptLoadRequest::Cancel();
-
-  mModuleScript = nullptr;
-  mReferrerScript = nullptr;
-  mModuleRequestObj = nullptr;
-}
-
 void ModuleLoadRequest::SetReady() {
   MOZ_ASSERT(!IsFinished());
 